app/app.cpp

```diff
@@ -3,5 +3,89 @@
 
 namespace vsite::oop::v3
 {
+	results::results(int c) {
+		count = c;
+		studNiz = new student[count];
+	}
 
+	void results::add(student s) {
+		studNiz[pos] = s;
+		pos++;
+	}
+
+	unsigned results::has_grade(int g) {
+		int studCount = 0;
+
+		for (int i = 0; i < count; i++) {
+			if (studNiz[i].grade == g) {
+				studCount++;
+			}
+		}
+		return studCount;
+	}
+
+	unsigned results::starts_with_letter(char c) {
+
+		int studCount = 0;
+
+		for (int i = 0; i < count; i++) {
+			if (tolower(studNiz[i].name[0]) == tolower(c)) {
+				studCount++;
+			}
+		}
+		return studCount;
+	}
+	results::~results()
+	{
+		delete[] studNiz;
+	}
+	
+	array::array(unsigned s, double value) {
```

Comment:
Probajte se navići inicijalizirati članove prije tijela (svih) konstruktora

`array::array(unsigned s, double value) : _size(s), nizArr(...) {`

Reply:
Nisam znao za takvo inicijaliziranje, pretpostavljam da je to sa zadnjeg predavanja ili mi je promaklo prije.